docs/dev/demo.md

```diff
@@ -0,0 +1,603 @@
+2024-01-29 11:01:10 info      Created successfully           action=create diffs={'+': {'content_types': {116}, 'key': 'cust_id', 'created': datetime.datetime(2021, 9, 9, 0, 0, tzinfo=datetime.timezone.utc), 'last_updated': None, 'type': 'text', 'label': 'Customer ID', 'description': None, 'required': False, 'filter_logic': 'exact', 'default': None, 'weight': 100, 'validation_minimum': None, 'validation_maximum': None, 'validation_regex': '^[A-Z]{3}\\d{2}$'}} dst=<NautobotAdapter "Nautobot"> flags=<DiffSyncFlags.NONE: 0> model=extras_customfield src=<NetBoxAdapter "NetBox"> status=success unique_id=e4fa5680-4657-50ea-837c-604617c7fc0d
+2024-01-29 11:01:10 debug     Attempting model create        action=create diffs={'+': {'content_types': {12, 13, 14, 7}, 'name': 'Unknown'}} dst=<NautobotAdapter "Nautobot"> flags=<DiffSyncFlags.NONE: 0> model=extras_status src=<NetBoxAdapter "NetBox"> unique_id=623acb70-22ce-5f69-bd7e-76bc921aa6c4
+2024-01-29 11:01:10 info      Created successfully           action=create diffs={'+': {'content_types': {12, 13, 14, 7}, 'name': 'Unknown'}} dst=<NautobotAdapter "Nautobot"> flags=<DiffSyncFlags.NONE: 0> model=extras_status src=<NetBoxAdapter "NetBox"> status=success unique_id=623acb70-22ce-5f69-bd7e-76bc921aa6c4
+
+...
+
+2024-01-29 11:01:28 debug     Attempting model create        action=create diffs={'+': {'created': datetime.datetime(2021, 4, 2, 0, 0, tzinfo=datetime.timezone.utc), 'last_updated': None, 'custom_field_data': None, 'name': 'South America', 'description': None}} dst=<NautobotAdapter "Nautobot"> flags=<DiffSyncFlags.NONE: 0> model=virtualization_clustergroup src=<NetBoxAdapter "NetBox"> unique_id=aa168442-0c73-5681-b8bd-243f5f93842f
+2024-01-29 11:01:28 info      Created successfully           action=create diffs={'+': {'created': datetime.datetime(2021, 4, 2, 0, 0, tzinfo=datetime.timezone.utc), 'last_updated': None, 'custom_field_data': None, 'name': 'South America', 'description': None}} dst=<NautobotAdapter "Nautobot"> flags=<DiffSyncFlags.NONE: 0> model=virtualization_clustergroup src=<NetBoxAdapter "NetBox"> status=success unique_id=aa168442-0c73-5681-b8bd-243f5f93842f
+2024-01-29 11:01:28 info      Sync complete                  dst=<NautobotAdapter "Nautobot"> flags=<DiffSyncFlags.NONE: 0> src=<NetBoxAdapter "NetBox">
+```
+
+### Summary
+
+The last part of the import process is to display a summary of the import [described in the user documentation](../user/summary.md).
+
+## Generated Mappings
```

Comment:
the data mappings are generated on the fly depending on the NetBox data and Nautobot models, isn't it? so it's something thatcan't be determined a prior by the versions without the real data, isn't it?

Reply:
Correct.